connectors/cli.py

```diff
@@ -101,15 +102,17 @@ def run(args):
         logger.info("Bye")
         return 0
 
-    service = SyncService(config, args)
-    coro = service.run()
+    services = [JobSchedulerService(config), JobExecutorService(config)]
     loop = get_event_loop(args.uvloop)
 
     for sig in (signal.SIGINT, signal.SIGTERM):
-        loop.add_signal_handler(sig, functools.partial(service.shutdown, sig))
+        for service in services:
+            loop.add_signal_handler(sig, functools.partial(service.shutdown, sig))
```

Comment:
we can't have several signal handlers for one signal. we need to have a single function, that iterates on services shutdown

Reply:
(but I think that's what you did in the other patch)

Reply:
Got it. Will make the change.

Reply:
I have done it in this PR: https://github.com/elastic/connectors-python/pull/337